doc/languages-frameworks/haskell.md

```diff
@@ -339,8 +339,9 @@ hoogle server -p 8080
 and navigate to http://localhost:8080/ for your own local
 [Hoogle](https://www.haskell.org/hoogle/). Note, however, that Firefox and
 possibly other browsers disallow navigation from `http:` to `file:` URIs for
-security reasons, which might be quite an inconvenience. See [this
-page](http://kb.mozillazine.org/Links_to_local_pages_do_not_work) for
+security reasons, which might be quite an inconvenience. Since version 5 hoogle
+server has a `--local` flag that solves the problem. For older versions see
```

Comment:
Can you describe how it solves the problem (by serving the files,) so people know the implications up front?